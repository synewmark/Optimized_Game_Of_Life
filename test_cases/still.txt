version https://git-lfs.github.com/spec/v1
oid sha256:291c8549ce54ff81bb6297d71a5235e938117e2be1629656351ec0542a397350
size 52
