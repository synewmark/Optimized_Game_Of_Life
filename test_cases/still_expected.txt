version https://git-lfs.github.com/spec/v1
oid sha256:8057c8e5c8f7a60d8f3b3de150aa3d113df8471c9fbfe9bc9b4480d77125d44d
size 40
